### DISCUSSION TITLE
Gov dashboard is not retrieving policy or securescore information

### DISCUSSION THREAD

**Discussion by thecloudman** [Azure/CCOInsights#305] [Q&A] [answered]
IS there a specific role in AAD the account signing into the custom connector requires to get Azure policy information and Secure Score information? My installation is logging in but is not retrieving any policy, secure score or resources from Azure.

**Comment by DrDonoso** [comment, marked as answer]
Hello,

If the user can see the policies in Azure, you should be able to see it in the Dashboard. Any role with reader permissions can read the policies.

**Comment by thecloudman** [comment]
Hi,

I have Global Reader but was only able to get one policy from one subscription. Seems like the connector isn’t fetching all subscriptions when connecting and only connecting to the first subscription it reads.

**Reply by DrDonoso** [reply]
Hello,

The dashboard is only fetching data from all the subscriptions to which the user has access.

- Can you see all the subscriptions in the subscription dashboard?
- Can you access the other subscriptions and see the data via the portal?
- Do the other subscriptions belong to the same or a different tenant?

Thanks.

**Comment by thecloudman** [comment]
Hello,

I have three tenants but in this case I am only signing into one tenant with 9 subscriptions. I can see all the subscriptions and all resources in them from the Azure portal.

May I suggest, possibly having a login option that request the Enrolment ID as part of the sign in process?

Thanks

**Reply by DrDonoso** [reply]
In the case you want to search in another tenant, you need a delegate resource management.
https://learn.microsoft.com/en-us/azure/lighthouse/how-to/view-manage-customers
